.github/workflows/build-push-image-ckan.yml

```diff
@@ -0,0 +1,46 @@
+---
+name: "Build CKAN image and push it to registry"
+
+on:
+  push:
+    branches:
+      - 'main'
```

Comment:
I think we need to adjust the branch name to "master" here (or alternatively change the default branch of the repo to "main" and adjust the PR base branch). This comment also applies to the other Workflow files.